app/templates/components/public/session-item.hbs

```diff
@@ -44,7 +44,7 @@
   <div class="content">
     {{#each session.speakers as |speaker|}}
       <div class="ui divider"></div>
-      <img alt="speaker" class="ui tiny avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
+      <img alt="speaker" class="ui avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
```

Comment:
You have just changed the size of image. For the given example even tiny class will give the appropriate image.

Reply:
oops ! i just missed that , sorry . So what is the difference between image i uploaded and the image that is showing blurred .

Reply:
As it's showing fine for 3-4 images i uploaded .

Reply:
`<img alt="speaker" class="ui tiny avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">`

why this line is there in code , there is no speaker icon  :   **if speaker.iconImageUrl speaker.iconImageUrl**

Reply:
There is

Reply:
Done :+1: